index.html

```diff
@@ -0,0 +1,193 @@
+                </div>
+
+                <div class="mobile_item-container" id="closemenu">
+                    <div class="mobile_item">
+                      <div id="mobile-close">
+                        <img onclick="clickToSwitch()" alt="close-icone-image" src="./image/exit.svg" />
+                      </div>
+                      <div class="mobile-menu">
+                        <ul class="mobile-list">
+                          <li><a  href="#" onclick="clickToSwitch()">Home</a></li>
+                          <li><a  href="./about.html" onclick="clickToSwitch()">About us</a></li>
+                          <li><a  href="#" onclick="clickToSwitch()">Mission</a></li>
+                          <li><a  href="#speaker" onclick="clickToSwitch()">Team</a></li>
+                          <li><a  href="#Partners" onclick="clickToSwitch()">Partners</a></li>
+                          <li><a  href="#" onclick="clickToSwitch()">CloudFlare</a></li>
+
+                        </ul>
+                    </div>
+                </div>
+            </div>
+          </nav>
+          </div>
+        </header>
+        </div>
+        <section class="headpart" id="head">
+          <h4 class="headpart1 head-h4 m-2">"Hello! Sharing world"</h4>
+          <h1 class="mobile pt-0">
+            Kenya Redcross<br>Fundraising concert<br>2015
+          </h1>
+          <h2 class="desktop head-h2 m-2">
+            Kenya Redcross<br>Fundraising concert<br>2015
+          </h2>
+          <p class="headpart-text bg-light">
+            We are organizing a consert for fundraising to feed people who are affected by the ongoing drought. 
+            Donate to support humanitarian work around Kenya. In times of crisis, we meet the urgent needs 
+            of women, men, young and the old. Help enable a rapid response to disasters. Your contribution can 
+            make a difference. All the funds which will be contributed will buy food items.
+          </p>
+          <h4 class="head-h4">LET'S CONNECT</h4>
+          <h4 class="head-h4 head-h4-strong" ><strong>FROM (MOND 8AM) ~ (SUN 17AM)</strong></h4>
+          <h4 class="head-h4 head-h4-strong">Lets fundraise</h4>
+        </section>
+     
+      <!-- Card section Mobile & Desktop -->
+      <section id="program" class="bg-dark">
+        <h4 class="text-center">Main Program</h4>
+        <div class="container pb-3">
+          <div class="row justify-content-md-center">
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/registration.webp" class="card-img-top bg-white" alt="icon">                
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Registration</h5>
+                  <p class="card-text">All the guests will register for attendance</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
```

Comment:
Kindly modify the program section to match the design guideline.

Please, let the background match that of the design. Also, make the cards look similar to the design by using matching font sizes, padding, and margins and maintaining equal width and height of the cards.

Check out other annotations in the image below.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192085124-1ce3f5c4-a195-4ce7-8dee-dbc3fa849315.png)
![image](https://user-images.githubusercontent.com/20554583/192085192-3d1d5be2-79f3-4901-8433-23241a455d80.png)

Design guideline: